sql/moz-fx-data-shared-prod/search_terms/sanitization_job_languages/view.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.search_terms.sanitization_job_languages`
+AS
+SELECT
+  started_at as job_start_time,
+  lang.key AS language_code,
+  CAST(lang.value AS int) AS search_term_count
```

Comment:
```suggestion
  SAFE_CAST(lang.value AS int) AS search_term_count
```

In case `lang.value` ever has a non-int value.